go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	cloud.google.com/go/pubsub v1.6.1
 	cloud.google.com/go/storage v1.10.0
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
-	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
```

Comment:
Do you know why it's gone?

Reply:
seems like previously we forgot to `tidy` stuff